components/dashboard/src/projects/Project.tsx

```diff
@@ -70,15 +69,6 @@ export default function () {
         }
     }
 
-    const branchContextMenu = (branch: Project.BranchDetails) => {
-        const entries: ContextMenuEntry[] = [];
-        entries.push({
-            title: "Rerun Prebuild",
```

Comment:
✋🏻 objection: this is removes the ability to trigger prebuilds entirely, no? but there are interesting cases to have it, for example if the configuration doesn't cover a particular branch.

Reply:
actually, if everyone is OK with letting this go, let's move forward. just want to make a point that we're loosing manual control over prebuilds if not configured for all branches.

Reply:
No, I've just inlined this `branchContextMenu` function for better readability. (It's easier to understand what a ContextMenu does when its options are also listed in the same location -- in a sense, ContextMenu options are also markup, and I don't like splitting markup into sub-functions all over the place).

The logic change is: Don't add this option all the time, but only when there is no prebuild, or when the prebuild has failed.